### PR TITLE
It's now ready to be merged ```brave-browser-beta```

### PR DESCRIPTION
**Pull request by NarutoXY**

### CODE REVIEW

File: packages/brave-browser-beta/brave-browser-beta.pacscript [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name="brave-browser-beta"
+pkgname="brave-browser-beta"
```
Comment (wizard-28) [inline packages/brave-browser-beta/brave-browser-beta.pacscript:2]:
```suggestion

```
Is this required?

File: packages/brave-browser-beta/brave-browser-beta.pacscript [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+version="1.27.81"
+url="https://github.com/brave/brave-browser/releases/download/v${version}/brave-browser-beta_${version}_amd64.deb"
+gives="brave-browser-beta"
```
Comment (wizard-28) [inline packages/brave-browser-beta/brave-browser-beta.pacscript:5]:
```suggestion

```
And this?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/brave-browser-beta/brave-browser-beta.pacscript (hunk 2 above)
Comment (NarutoXY) [reply packages/brave-browser-beta/brave-browser-beta.pacscript:4]: I'm currently using it to remove deb packages using apt

**Review by wizard-28 — APPROVED** [review]

**Comment by wizard-28** [comment]
@NarutoXY #69 

*Nice*

**Pull request merged**